sdk/cognitivelanguage/azure-ai-language-questionanswering/swagger/README.md

```diff
@@ -68,6 +68,64 @@ title: QuestionAnsweringAuthoringClient
 
 ## Customizations
 
+### General customizations
+
+```yaml
+# Define HTTP 200 responses for LROs to document result model.
+directive:
+- where-operation: QuestionAnsweringProjects_DeleteProject
+  transform: |
+    $.responses["200"] = {
+      description: "Project delete job status.",
+      schema: {
+        "$ref": "#/definitions/JobState"
+      }
+    };
+- where-operation: QuestionAnsweringProjects_DeployProject
+  transform: |
+    $.responses["200"] = {
+      description: "Deploy job state.",
+      schema: {
+        "$ref": "#/definitions/ProjectDeployment"
+      }
+    };
+- where-operation: QuestionAnsweringProjects_Import
+  transform: |
+    $.responses["200"] = {
+      description: "Gets the status of an Import job.",
+      schema: {
+        "$ref": "#/definitions/JobState"
+      }
+    };
+- where-operation: QuestionAnsweringProjects_UpdateQnas
+  transform: |
+    $["x-ms-pageable"] = {
+      "nextLinkName": "nextLink",
+      "itemName": "value"
+    };
```

Comment:
The x-ms-pageable addition should be updated in the swagger. Discussion in https://github.com/Azure/azure-sdk-for-python/issues/25304